Hunting Queries/Microsoft 365 Defender/TVM/Microsoft Defender AV details.yaml

```diff
@@ -0,0 +1,44 @@
+id: FEE07B7B-E704-4216-BBFC-3B1344136E07
+name: Microsoft Defender AV details
+description: |
+  // Author: Sonali Meshram
+  // @someshram
+  // ------------------------
+  // 1.	A list of all devices with below AV information
+  // 2.	This query will identify the Microsoft Defender Antivirus Security Intelligence version, Security Intelligence up to date value, Engine version, Engine up to date value,  Product version (aka Platform version),Product (aka Platform) up to date value, Security Intelligence publish/build timestamp, Security intel refresh timestamp
+requiredDataConnectors:
+- connectorId: MicrosoftThreatProtection
+  dataTypes:
+  - DeviceTvmInfoGathering
+query: |
```

Comment:
Please add Tactics and relevant technique and if it is not applicable, just leave it as blank. i.e 

tactics:[]
relevantTechniques:[]

Reply:
resolved